jekyll/index.html

```diff
@@ -216,17 +216,15 @@ <h1 class="section-heading">
         <h2>Forum</h2>
         <h3>Threaded communication</h3>
         <a href="https://forum.nim-lang.org">
-          <i class="fa fa-comments" aria-hidden="true"></i>
-          https://forum.nim-lang.org
+          <i class="fa fa-comments" aria-hidden="true"></i><nobr>forum.nim-lang.org</nobr>
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr

> This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user.